interfaces/Libraries.ts

```diff
@@ -37,6 +45,10 @@ interface ModelData {
 	 * this is transformers-specific
 	 */
 	autoArchitecture: string;
+	/**
+	 * this dictionary has useful information about the model configuration
+	 */
+	config: Obj
```

Comment:
```suggestion
	config: Record<string, any>;
```

you can type this like that which is built-in (and remove the `Obj` above)